app/routes/projects/index.tsx

```diff
@@ -678,33 +678,33 @@ export default function Projects() {
               ) : (
                 <Table size="small">
                   <TableHead>
-                    <TableRow>
-                      <TableCell sx={{ fontWeight: "bold" }}>Name</TableCell>
-                      <TableCell sx={{ fontWeight: "bold" }}>Owner</TableCell>
-                      <TableCell sx={{ fontWeight: "bold" }}>Tier</TableCell>
-                      <TableCell sx={{ fontWeight: "bold" }}>Status</TableCell>
-                      <TableCell sx={{ fontWeight: "bold" }}>
-                        Contributors
-                      </TableCell>
-                      <TableCell sx={{ fontWeight: "bold" }}>Likes</TableCell>
-                      <TableCell sx={{ fontWeight: "bold" }}>Skills</TableCell>
+                  <TableRow>
+                      <TableCell>Name</TableCell>
+                      <TableCell>Tier</TableCell>
+                      <TableCell>Status</TableCell>
+                      <TableCell>Contributors</TableCell>
+                      <TableCell>Likes</TableCell>
+                      <TableCell>Skills</TableCell>
+                      <TableCell>Resources</TableCell>
                     </TableRow>
                   </TableHead>
                   <TableBody>
                     {projects.map((item, i) => (
                       <TableRow key={i}>
-                        <TableCell>
-                          <Link to={`/projects/${item.id}`}>{item.name}</Link>
-                        </TableCell>
-                        <TableCell>
-                          {item.preferredName} {item.lastName}
-                        </TableCell>
-                        <TableCell>{item.tierName}</TableCell>
-                        <TableCell>{item.status}</TableCell>
-                        <TableCell>{item.projectMembers}</TableCell>
-                        <TableCell>{item.votesCount}</TableCell>
-                        <TableCell>{item.searchSkills}</TableCell>
-                      </TableRow>
+                      <TableCell><a href={`/projects/${item.id}`} style={{ color: 'inherit', textDecoration: 'none' }}>{item.name}</a></TableCell>
+                      <TableCell>
+                        <a href="https://wizeline.atlassian.net/wiki/spaces/wiki/pages/3075342381/Innovation+Tiers"
```

Comment:
I'm thinking we could instead make the heading cell the link to what are innovation tiers, and make each project individual tier cell link to this page but filtered by that innovation tier. Similar to the innovation tier links inside a project view, like https://labs.wizeline.io/projects/ffc937dd-fbca-4855-9e2d-e895fc434332 that points to https://labs.wizeline.io/projects?tier=Tier%201

Same for project status and each skill. To link to the project list page filtered by status or skill.

Reply:
Fixed